Fix big-endian build breakage in protolib.
Patch status: submitted to author via email.

Signed-off-by: Gustavo Zacarias <gustavo@zacarias.com.ar>

diff -Nura norm-1.5r6.orig/protolib/include/protoSpace.h norm-1.5r6/protolib/include/protoSpace.h
--- norm-1.5r6.orig/protolib/include/protoSpace.h	2016-03-08 19:48:21.326357963 -0300
+++ norm-1.5r6/protolib/include/protoSpace.h	2016-03-08 19:48:46.647234075 -0300
@@ -89,7 +89,7 @@
                 }
 #else               
                 ProtoTree::Endian GetEndian() const
-                        return ProtoTree::ENDIAN_BIG;
+                    {return ProtoTree::ENDIAN_BIG;}
                 void SetNode(Node* theNode)
                     {memcpy(key+sizeof(double), &theNode, sizeof(Node*));}
                 Node* GetNode() const
